pkg/ecctl/init.go

```diff
@@ -243,6 +259,26 @@ func printConfig(writer io.Writer, v *viper.Viper) error {
 	return enc.Encode(c)
 }
 
+func askInfraSelection(cfg *Config, scanner *input.Scanner, writer, errWriter io.Writer) error {
+	infraChoiceRaw := scanner.Scan(hostChoiceMsg)
+	fmt.Fprintln(writer)
+	infraChoice, err := strconv.Atoi(infraChoiceRaw)
+	if err != nil {
+		return err
+	}
+
+	cfg.Host = "https://api.elastic-cloud.com"
+	switch infraChoice {
+	case eceInfraChoice:
+		cfg.Host = scanner.Scan(hostMsg)
+	case essInfraChoice:
```

Comment:
Maybe we can print a line stating the URL which will be used? It seems a bit empty when you select that you get no confirmation given that the other 2 options (Invalid or ECE, do have some more info).